gradle.properties

```diff
@@ -1 +1 @@
-VERSION=0.1.0
\ No newline at end of file
+VERSION=0.1.1
```

Comment:
I think this deserves a 0.2.0, no?

Reply:
It had no API changes, so I thought bumping the `patch` is enough.